### PR TITLE
Some fixes in spelling and language shit and stuffz

### PR DESCRIPTION
**Pull request by MDMCK10**
title says all

### CODE REVIEW

**Comment by MDMCK10** [comment]
k, all ready to merge

File: http/lang/en/about.lang [hunk 1]
```diff
@@ -2,7 +2,7 @@ about			About
 
 contact			Contact us
 
-contact_text			You can contact us by email at %s, by private message on the game, by Facebook or Twitter.
+contact_text			You can contact us by email at %s, by private message on the game, or by Facebook or Twitter.
```
Comment (Shakadak) [inline http/lang/en/about.lang]: I think that `, by Facebook, or by Twitter.` is a better phrasing.

Comment (jeuxjeux20) [inline http/lang/en/about.lang]: Indeed :/

File: http/lang/en/login.lang [hunk 2]
```diff
@@ -2,10 +2,10 @@ connection			Log in
 error_farmer_banned			Your account has been banned
 error_farmer_inactive			Your account is not activated. To confirme the creation of your account, access your email and click on the link provided.
 error_invalid			Your IDs are invalid.
-forgot_password			Password forgotten ?
+forgot_password			Forgot password?
```
Comment (Shakadak) [inline http/lang/en/login.lang]: If you want to use the active voice, I would suggest `Forgot your password?`.

**Comment by MDMCK10** [comment]
ok is this fine now

File: http/lang/en/about.lang [hunk 3]
```diff
@@ -2,7 +2,7 @@ about			About
 
 contact			Contact us
 
-contact_text			You can contact us by email at %s, by private message on the game, by Facebook or Twitter.
+contact_text			You can contact us by email at %s or by private message on the game, and by Facebook or Twitter.
```
Comment (Shakadak) [inline http/lang/en/about.lang]: I think this one is getting even more awkward. It is valid, but just listing all the possibilities without grouping them seems more appropriate. Eg. `You can contact us via email at %s, private message on the game, Facebook, or Twitter.` That's just my opinion though.

**Comment by MDMCK10** [comment]
what about now
._.

**Pull request merged**